schema.sql

```diff
@@ -3,7 +3,7 @@ CREATE TABLE books(
   id SERIAL PRIMARY KEY,
   title VARCHAR(255),
   author VARCHAR(255),
-  image_url VARCHAR(255),
+  image_url TEXT,
```

Comment:
Why text?  Seems a little too big for a URL.  Check out this interesting SO post: http://stackoverflow.com/questions/417142/what-is-the-maximum-length-of-a-url-in-different-browsers